recipes/gluonts/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "gluonts" %}
+{% set version = "0.11.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gluonts-{{ version }}.tar.gz
+  sha256: c5db67304853db450620edf2a5bf248eb7e6ee65e4290641bf725e4a2e7d00db
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - sphinx
+    - click
+    - mypy
+    - black
+    - pip
```

```suggestion
  host:
    - python
    - pip
```
AFAICS these aren't required to build the package?

`click` seems that it might be a run-time dependency?